src/EventTickets/Migrations/CreateEventsTable.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Give\EventTickets\Migrations;
+
+use Give\Framework\Database\DB;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+/**
+ * @unreleased
+ */
+class CreateEventsTable extends Migration {
+    /**
+     * @inheritdoc
+     */
+    public static function id() {
+        return 'give-events-create-events-table';
+    }
+
+    public static function title() {
+        return 'Create give_events table';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public static function timestamp() {
+        return strtotime( '2022-01-29 00:00:00' );
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function run() {
+        global $wpdb;
+
+        $table   = $wpdb->give_events;
+        $charset = DB::get_charset_collate();
+
+        $sql = "CREATE TABLE $table (
+			id INT UNSIGNED NOT NULL,
+			form_id INT NULL,
```

Comment:
We don't need to assign forms to events because we currently have a one-to-many relationship, so it may not be necessary.

Reply:
Good point - I'll remove the `form_id` property and column.

Reply:
Resolved, see [8b39068](https://github.com/impress-org/givewp/pull/7213/commits/8b39068012f9ec06c177574ec2c5779f526531d4).